ark/analysis/spatial_analysis_test.py

```diff
@@ -185,6 +185,30 @@ def _make_expression_matrix(enrichment_type):
         return all_patient_data_neg
 
 
+def _make_neighborhood_matrix():
+    col_names = {0: 'SampleID', 1: 'cellLabelInImage', 2: 'feature1', 3: 'feature2'}
+    neighbor_counts = pd.DataFrame(np.zeros((100, 5)))
+    neighbor_counts = neighbor_counts.rename(col_names, axis=1)
+
+    neighbor_counts.iloc[0:50, 0] = "Point1"
```

Comment:
Let's simplify this is a bit. There should be three populations. One that is low for both features, one that is high for both features, and one that has one low and one high. 

Each FOV should have two of the three populations. Then we can check that three is what's predicted from the silhoutte score

Reply:
If I'm understand the population distribution correctly, say fov1 has 90 cell labels. Let's say we divide this into two partitions of 45 cells each. So we pick one population, say low, for the first 45 cells and another population, say high, for the next 45 cells. We do the same thing for fov2 as well, except we can populate with both high and "low and high" distributions.

Reply:
yup exactly